packages/wallet/src/KeyManagement/TrezorKeyAgent.ts

```diff
@@ -0,0 +1,129 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { AuthenticationError, TransportError } from './errors';
+import { Cardano, NotImplementedError } from '@cardano-sdk/core';
+import { CommunicationType, KeyAgentType, SerializableTrezorKeyAgentData, SignBlobResult } from './types';
+import { KeyAgentBase } from './KeyAgentBase';
+import TrezorConnect, { Features } from 'trezor-connect';
+
+export type TrezorKeyAgentProps = Omit<SerializableTrezorKeyAgentData, '__typename'>;
+
+export interface CreateWithDevice {
```

Comment:
Let's rename all 'Device' to 'Trezor' since there are now 2 hw key agent device types. Maybe review LedgerKeyAgent type names too to make them more specific, etc. Device->Ledger, Transport->LedgerTransport
```suggestion
export interface CreateTrezorKeyAgentProps {
```

Reply:
@mkazlauskas I wanted to unify naming to prevent breaking changes in LW but I agree that it should be better :) 
I will do that

Reply:
Done!